web/src/upload.js

```diff
@@ -129,7 +130,11 @@ export function get_item(key, config, file_id) {
 export function hide_upload_banner(uppy, config, file_id) {
     get_item("upload_banner", config, file_id).remove();
     if (uppy.getFiles().length === 0) {
-        get_item("send_button", config).prop("disabled", false);
+        if (config.mode === "compose") {
+            compose_validate.update_upload_in_progress_status(false);
+        } else {
+            get_item("send_button", config).prop("disabled", false);
+        }
```

Comment:
I guess we don't yet have a version of `update_message_too_long_status` for message editing (this is https://github.com/zulip/zulip/issues/25271), but presumably this bug applies there. So I'm OK with merging this without that addressed if we just make a note to fix this while we're at it.

Reply:
We actually have a `maxlength` set for the edit textarea, so this isn't buggy for the edit case

Reply:
OK